tests/e2e/scale/test_cephfs_many_files.py

```diff
@@ -49,9 +49,12 @@ def add_million_files(pod_name, ocp_obj):
             test_file_list.append(fname.split(os.sep)[-1])
     tmploc = ntar_loc.split("/")[-1]
     run_cmd(f"tar cfz {tarfile} -C {new_dir} .")
-    ocp_obj.exec_oc_cmd(
-        f"rsync {ntar_loc} {pod_name}:{constants.MOUNT_POINT}", timeout=300
-    )
+    try:
+        ocp_obj.exec_oc_cmd(
+            f"rsync {ntar_loc} {pod_name}:{constants.MOUNT_POINT}", timeout=300
```

Comment:
@wusui Please check either you can use this function to rsync data https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/ocp.py#L1044